src/test/java/utils/DateAndTimeUtils.java

```diff
@@ -0,0 +1,34 @@
+package utils;
+
+import java.text.SimpleDateFormat;
+import java.util.Date;
+public class DateAndTimeUtils {
+
+
+    // Get the current date and time as a formatted string
+    public static String getCurrentDateTime(String format) {
+        SimpleDateFormat dateFormat = new SimpleDateFormat(format);
+        return dateFormat.format(new Date());
+    }
+
+    // Get the current date and time as a formatted string using a default format
+    public static String getCurrentDateTime() {
+        return getCurrentDateTime("yyyy-MM-dd HH:mm:ss");
+    }
+
+    // Add or subtract days to/from a given date
+    public static String addDaysToCurrentDate(String format, int days) {
```

Comment:
description says "Add or subtract days" but method only adds. Can you add a logic to subtract days as well?